code/game/gamemodes/dynamic/antag_rulesets.dm

```diff
@@ -0,0 +1,171 @@
+#define MINDSHIELDED_JOBS list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Trans-Solar Federation General")
+
+/datum/ruleset
```

Comment:
All the var-level docs are good but a top-level doc comment would be useful here, especially since the name of the type doesn't provide any context as to what "ruleset" is referring to